titan-compiler/checker.lua

```diff
@@ -367,6 +404,32 @@ check_var = function(var, errors)
     end
 end
 
+
+local function check_exp_callfunc_builtin(exp, errors)
```

Comment:
Could we move this function to the top of the file?
I think it is strange to have it between others `check_` functions (despite it is only used by call_exp).

Reply:
I would prefer keeping these closer to check_exp, since that is the only place where they will be used.